homepage/tests.py

```diff
@@ -0,0 +1,3 @@
+def test_trivial():

```

Comment:
@LeonRabinovich in the next PR when you add a real tests please don't forget to remove this one